apps/rush/src/utilities/PackageReviewChecker.ts

```diff
@@ -3,47 +3,59 @@
 
 import {
   IPackageJson,
+  ApprovedPackagesPolicy,
   RushConfiguration,
   RushConfigurationProject,
-  PackageReviewConfiguration,
   Utilities
 } from '@microsoft/rush-lib';
 
-export default class PackageReviewChecker {
-  private _rushConfiguration: RushConfiguration;
-  private _packageReviewConfiguration: PackageReviewConfiguration;
-
-  constructor(rushConfiguraton: RushConfiguration) {
-    this._rushConfiguration = rushConfiguraton;
-    this._packageReviewConfiguration = PackageReviewConfiguration.loadFromFile(rushConfiguraton.packageReviewFile);
-  }
+export class PackageReviewChecker {
```

Comment:
@nickpape-msft

My understanding was that everyone thinks "export default" was a poor choice for our team standard, including jonguy who argued for it in the first place.  The reason is that it makes the "import" statements inconsistent, and it makes it super easy to accidentally import the wrong class (e.g. `import Settings from './ISettings';` can produce some really strange error messages).

I've been thinking about what's the right way to update the standard and migrate our code.  However, Nick said he just stopped using "export default" in the code he's writing.

This is out of scope for this PR, but I'll see if I can start a discussion about it.

---
In reply to: [116284380](https://github.com/Microsoft/web-build-tools/pull/181#discussion_r116284380) [](ancestors = 116284380)